docs/source/detsim.rst

```diff
@@ -1,5 +1,299 @@
+the simulated event. Since each of the emitted photons is also
+simulated, the final result of the simulation will be a number of
+photons arriving at the photo-sensors. The output of the simulation is
+given as zero-suppressed time histograms with the photo-electrons
+generated in each sensor (left figure). This simulation mode is
+both CPU-time-consuming and memory demanding, with a time-scale on the
+order of 2 min to simulate a 41.5 keV :math:`^{83m}`\ Kr event and > 1
+hour for a :math:`Q_{\beta\beta} \sim 2.5` MeV event. The advantage of
+this simulation mode is its extreme detail and that its outcome is the
+actual signal measured at the light sensors.
+
+**Fast-Simulation:** In this mode the simulation is stopped after the
+creation of the ionization electrons and scintillation photons. The
+ionization electrons and scintillation photons are produced by the
+direct propagation of primary particles, or by secondary particles
+emitted by the primaries, like delta electrons or bremsstrahlung that
+subsequently interact. Once they are created, the simulation stops, and
+neither drift nor light propagation is simulated. The ionization
+electrons and scintillation photons created are collected in voxels of a
+certain volume (usually around 1x1x1 mm\ :math:`^3` voxels). Each voxel
+represents an energy deposition, a so-called *hit*, and is characterized
+by a 3D position :math:`\boldsymbol{r} = (x, y, z)`, the energy
+:math:`E` required to produce them, and the delay time :math:`t` from
+the start of the event (right figure). The
+collection of hits for a given even is saved, defining a *track* in the
+detector’s active volume. The fast-simulation is very fast compared with
+the full-simulation mode, on the :math:`\mu s` scale. However, since the
+simulation is stopped we do not know the signal that would be measured
+at the sensors.
+
+.. image:: images/detsim/fullsim_output.png
+   :width: 40%
+
+.. image:: images/detsim/fastsim_output.png
+   :width: 40%
+
+
+.. _Detsim input:
+
+Input
+-----
+
+Detsim input are NEXUS fast simulation tracks, these files must contain only the following non-empty information.
+
+ * ``/MC/hits``
+
+.. _Detsim output:
+
+Output
+------
+
+ * ``/MC/``: the data in the nexus input file is copied in the output file
+ * ``/Run/event_map``: mapping between nexus event id and IC event id. This mapping is needed because **detsim** might split nexus events into multiple IC events
+ * ``/Run/events``: IC event number and simulated timestamps based on **rate** configuration parameter
+ * ``/Run/runInfo``: run number of each event
+ * ``/pmtrd``: the PMT waveforms for each PMT
+ * ``/sipmrd``: the SiPM waveforms for each SiPM
+ * ``/Filters/active_hits``: nexus events without hits inside the active volume are filtered
+
+.. _Detsim config:
+
+Config
+------
+
+.. container::
+    :name: tab:detsim-parameters
+    
+    .. table:: List of **detsim** parameters used at each stage of the algorithm.
```

Comment:
Add all the parameters used in detsim following the guidelines from the other cities, https://pr18-sw-docs.readthedocs.io/en/latest/irene.html#config. Including the parameter name that will be used in the config, plus the type of variable.